### ISSUE TITLE
[Select] Accessibility test for Select component fails due to unrecognized reason.

### ISSUE THREAD

**Issue by blackswanny** [uber-web/baseui#424]
Accessibility test for Select component in src/select/e2e.js fails due to unrecognized reason.
aXe core code doesn't reflect any violations of Accessibility, however nightwatch plugin, which utilizes it, returns error back as no results come from aXe.

Need to add it back.
```js
  it('passes basic a11y tests', browser => {
    goToUrl({
      suite,
      test: scenarios.SELECT,
      browser,
    })
      .initAccessibility()
      .waitForElementVisible(selectors.selectInput, 1000)
      .assert.accessibility('html', {
        rules: {
          'aria-labelledby': {
            enabled: false,
          },
        },
      });
  });
```
